common/src/main/java/bisq/common/crypto/KeyStorage.java

```diff
@@ -97,6 +97,7 @@ public String toString() {
 
     @Inject
     public KeyStorage(@Named(KEY_STORAGE_DIR) File storageDir) {
+        storageDir.mkdirs();
```

Comment:
the CI error came from a missing key directory on CI. I now changed the keystorage to create the directory if its missing. not sure if that has any downsides.